.github/workflows/js-css-linting.yml

```diff
@@ -57,12 +77,19 @@ jobs:
 
         steps:
             - uses: actions/checkout@v3
-            - name: Setup node version and npm cache
+            - name: Cache node_modules
+              id: cache-node-modules
+              uses: actions/cache@v3
+              env:
+                  cache-name: cache-node-modules
+              with:
+                  path: node_modules
+                  key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+
+            - name: Setup node version
               uses: actions/setup-node@v3
               with:
                   node-version-file: '.nvmrc'
-                  cache: 'npm'
-            - name: Install Node dependencies
-              run: npm ci --no-optional
```

Comment:
Same thing here! After removing the `Install Node dependencies` step from the `CSSLintingCheck` job, I think it's safe to remove the `Cache node_modules` & `Setup node version` as well!

Reply:
See[ comment above](https://github.com/woocommerce/woocommerce-blocks/pull/6404/files/6bd2555e729593a171b59d1a6b3c2037560481cf#r880346473)